dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/StoredCompanyRepository.kt

```diff
@@ -47,7 +50,69 @@ interface StoredCompanyRepository : JpaRepository<StoredCompanyEntity, String> {
             "WHEN lower(max(alternativeName)) LIKE :#{#searchFilter.searchStringLower}% THEN 4 ELSE 5 END) ASC, " +
             "company.companyName ASC",
     )
-    fun searchCompanies(@Param("searchFilter") searchFilter: StoredCompanySearchFilter): List<StoredCompanyEntity>
+    fun searchCompanies(@Param("searchFilter") searchFilter: StoredCompanySearchFilter, pageable: Pageable):
+        List<StoredCompanyEntity>
```

Comment:
This pagination stuff is new to me, but after a short google research it seems to me that this is currently wrong.
The correct way would be that the function returns a `Page<StoredCompanyEntity>` object if I understand it right. Wouldn't it?

Reply:
Well, what means correct. Can it be incorrect if it compiles and works? I'll check if there might be a performance issue.

Reply:
A pageable seems to have additional properties we do not need. But accessing the date seems to work via a list returned by a method as well.

Reply:
Es ist nicht falsch, aber es scheint keinen Nutzen zu stiften.   Also ich meine wir können es dann auch rausnehmen.

Reply:
Wurde rausgenommen. Daher schließe ich das 👍